contracts/programs/bubblegum/src/lib.rs

```diff
@@ -321,6 +375,132 @@ pub fn get_instruction_type(full_bytes: &[u8]) -> InstructionName {
     }
 }
 
+fn assert_enough_mints_to_approve(
+    total_mint_capacity: u64,
+    num_mints_approved: u64,
+    num_minted: u64,
+    to_approve: u64,
+) -> Result<()> {
+    let remaining_mints_to_approve = total_mint_capacity.saturating_sub(num_mints_approved);
+    let remaining_mints = total_mint_capacity.saturating_sub(num_minted);
+    msg!(&format!(
+        "{} {} {}",
+        remaining_mints_to_approve, remaining_mints, to_approve
+    ));
+    if to_approve > remaining_mints_to_approve || to_approve > remaining_mints {
+        return Err(BubblegumError::NotEnoughMintCapacity.into());
```

Comment:
Nit: `InsufficientMintCapacity` is a better error code

Reply:
addressed